.github/workflows/auto-diff-k8s-ci.yaml

```diff
@@ -128,12 +128,14 @@ jobs:
       - name: get latest kind node version
         run: |
           bash ./test/scripts/update-kindnode-version.sh ${{ env.MINIMUM_K8S_VERSION }} ${{ env.K8S_MATRIX_FILE_PATH }}
-          if [ $? -eq 0 ]; then
+          echo "status code: $?"
+          if [ $? -ne 0 ]; then
```

Comment:
this is a new bug 
"$?" in line 132 is for the result of line 131 "echo"

Reply:
this should not fail the CI 

if ! ./test/scripts/update-kindnode-version.sh ... ; then

Reply:
```
          bash ./test/scripts/update-kindnode-version.sh ${{ env.MINIMUM_K8S_VERSION }} ${{ env.K8S_MATRIX_FILE_PATH }}
          if [ $? -ne 0 ]; then
            echo "./test/scripts/update-kindnode-version.sh script failed"
            echo "SKIP_CREATE_PR=true" >> $GITHUB_ENV
            exit 0
          fi
```
yes, I have modified it so that now it does not cause CI failures, it just prints the error log.